.github/workflows/ci.yml

```diff
@@ -17,6 +17,6 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.1
```

Comment:
oh, we've got 4.1.1 now? what do we do with all the others which are at 4.1.0?

Reply:
Nope it should be 4.1.0

But we have the same situation with v4.0.0 and v4.1.0 and the only difference is the BlueOak license

So for now let’s try to use the latest versions but it’s fine if some repos don’t